src/components/ui/Skeleton/tests/Skeleton.test.tsx

```
@@ -0,0 +1,45 @@
+
+        const spanElements = container.querySelectorAll('span');
+        expect(spanElements.length).toEqual(0);
+
+        const divElements = container.querySelectorAll('div');
+        expect(divElements.length).toEqual(1);
+        expect(divElements[0]).toHaveTextContent('Test Children Component');
+    });
+    
+    it('renders with span wrapper on loading is true', () => {
+        const { container } = render(
+            <Skeleton loading={true} children={mockChildren}></Skeleton>
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)